keps/sig-storage/1710-selinux-relabeling/README.md

```diff
@@ -572,8 +592,8 @@ _This section must be completed when targeting beta graduation to a release._
 
   - [x] Metrics
     - All `errors_total` metrics below cover real errors when a Pod can't start.
-      It applies to `ReadWriteOncePod` volumes.
-    - All `warnings_total` metrics below cover **future** errors that would appear if this feature was extended to all volumes.
+      It applies to `ReadWriteOncePod` volumes when only `SELinuxMountReadWriteOncePod` feature gate is enabled or all volumes when `SELinuxMount` feature gate is enabled.
```

Comment:
"To enable `SELinuxMount` feature gate, `SELinuxMountReadWriteOncePod` **must** be enabled too."

So should this be changed to "all volumes when both `SELinuxMountReadWriteOncePod` and `SELinuxMount` feature gates are enabled"?  If only `SELinuxMount` is enabled, that's treated as none of these feature gates are enabled?

Reply:
Good point, I rephrased and reformatted this to be 100% clear.